<?xml version="1.0"?>
<launch>

    <arg name="n"/>

    <include file="$(dirname)/$(arg n)Vehs/$(env HOSTNAME).launch"/>

    <node name="vehicle" pkg="wp3_tests" type="VehicleP2P.py" output="screen">
        <param name="standby_freq" value="5"/>
    </node>

</launch>
